### DISCUSSION TITLE
Updated wormhole traversal challenge

### DISCUSSION THREAD

**Discussion by cantordust** [esa/SpOC2#16] [Announcements]
Dear SpOC competitors,

We would like to inform you that the wormhole network traversal challenge is back online. The UDP, the problem description and the example have been updated here in this repository as well as on the Optimise platform. In a nutshell, the update introduces a penalty for traversing the same node multiple times by multiplicatively increasing the variance of all edges of such nodes.

Please note that due to the nature of the update, we would like to invite all teams that had submitted solutions to do so again.

As usual, let us know if you have any questions, comments and feedback. May the event horizon be with you!

Alex on behalf of the ACT.

**Comment by dietmarwo** [comment]
Dear SpoC Team,
the change makes sense. The task is more balanced now. 

Question to the wormhole leaderboard:
- the first Team (theEs) has 1,405,234,192,818.721. Is this correct (maximal sum of the variances)?
The upload was after the update, so I would expect the teams to avoid multiple visits now which should lead
to much lower scores (around 0.4).

Question to the rovers leaderboard:
- the first Team (theEs) has rank 2 here (if you click on the team), should be rank 1.

**Reply by cantordust** [reply]
Dear @dietmarwo ,

Thank you for reaching out! Regarding your first question, we cannot say at this stage what the other teams are doing, but the code is working as expected.

The second issue is being addressed right now and should be corrected shortly. Thank you for keeping a keen eye on the leaderboards!

Cheers,

Alex

**Reply by dietmarwo** [reply]
Thanks a lot for fixing. In the meantime I found some time to check the wormhole code myself (fcmaes), everything is fine, there are many teams in the 3 digit range now.
